### PR TITLE
Website-building

### PR DESCRIPTION
**Pull request by adel-gu**
## Accomplishments  💪
**In this PR I did the following tasks:**
- Setup project repo, and Linter for HTML, CSS, and JavaScript.
- Using Bootstrap I built both screen versions, Desktop, and mobile.
- Interactive menu button for the mobile version.
- JavaScript Generating Markup for both Program section and Speakers section cards.

### CODE REVIEW

File: interfaces/about.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon btn"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav gap-md-5 ms-auto mb-2 mb-lg-0 text-center">
+              <li class="nav-item">
+                <a class="nav-link current" aria-current="page" href="#"
+                  >About</a
+                >
```
Comment (jasonfelice) [inline interfaces/about.html:78]: - [x]  [OPTIONAL]  When the user is on the about page, there's no way to go back to the home page, especially on mobile devices. I highly suggest making sure this link navigates to the home page instead. 😄

File: interfaces/about.html [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                <p>DevBox Global 2018 in England</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- PARTNER SECTION -->
+    <section id="partners" class="bg-dark d-md-none">
```
Comment (jasonfelice) [inline interfaces/about.html]: - [x] Kindly add a partners section also for a desktop screen on the about page. Currently, it's only visible in the mobile version.
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        <div class="container-md text-center">
+          <h2>Featured Speakers</h2>
+          <hr class="separator" />
+          <ul class="cards"></ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- PARTNER SECTION -->
+    <section id="partners" class="bg-dark d-md-none">
```
Comment (jasonfelice) [inline index.html]: - [x] Kindly add a partners section also for a desktop screen on the index page. Currently, it's only visible in the mobile version.

image

File: interfaces/about.html [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+              <div class="past-conf-1 mb-3 mb-lg-0">
+                <strong>2015</strong>
+                <p>DevBox Global 2015 in France</p>
+              </div>
+            </div>
+            <div class="col-12 col-lg-6">
+              <div class="past-conf-2">
+                <strong>2018</strong>
+                <p>DevBox Global 2018 in England</p>
+              </div>
```
Comment (jasonfelice) [inline interfaces/about.html]: - [x] Kindly fix the alignment of these cards. (On resolution > 1399px) 😄 

image

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @adel-gu, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
- [x] Please submit a video presentation link in the PR description.

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: interfaces/about.html (hunk 1 above)
Comment (adel-gu) [reply interfaces/about.html:78]: Hi @jasonfelice, Thanks for making it easy, I was thinking about this the whole day 😅, totally agree it's hard to navigate to the home page so am just gonna add a home page link.

File: interfaces/about.html (hunk 4 above)
Comment (adel-gu) [reply interfaces/about.html]: That's a nice catch 👏👏👏, I really test it on different screen sizes and I miss that resolution, Thank you.

File: index.html (hunk 3 above)
Comment (adel-gu) [reply index.html]: I'll do so🐱‍🏍, thank you 👏.

**Review by emmiiorji — APPROVED** [review]
Hi @adel-gu, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats


## Good Highlights ✔

- Requested changes have been implemented
- Good commit messages
- Adherence to GitHub flow

## Optional suggestions

- Currently, while on the about page, a click on the about mobile menu does not hide the mobile menu. You may want to look into that.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adel-gu** [comment]
Hi @emmiiorji 🙋‍♂️

Thank you for the about menu button comment, and yes sure I'll do so 🐱‍🏍.

**Pull request merged**